Sources/StreamChatUI/CommonViews/Attachments/AttachmentPreview/ImageAttachmentPreview.swift

```diff
@@ -12,14 +12,32 @@ public struct ImageAttachmentPreview: AttachmentPreview {
     public let name: String?
     /// The size of the attachment.
     public let size: Int64
+    /// The local url of the image attachment.
+    public let localURL: URL
 
-    public init(image: UIImage, name: String? = nil) {
+    public init(image: UIImage, localURL: URL, name: String? = nil) {
         self.image = image
         self.name = name
+        self.localURL = localURL
         if let imageData = image.imageData, !imageData.isEmpty {
             size = Int64(imageData.count) / 1000
         } else {
             size = 0
         }
     }
 }
+
+// TODO:
+public struct DocumentAttachmentPreview: AttachmentPreview {
+    public let image: UIImage
+    public let name: String?
+    public let size: Int64
+    public let localURL: URL
+
+    public init(image: UIImage, localURL: URL, name: String, size: Int64) {
+        self.image = image
+        self.name = name
+        self.size = size
+        self.localURL = localURL
+    }
+}
```

Comment:
Ignore this bit, this will be changed, when https://github.com/GetStream/stream-chat-swift/pull/1071 is merged